guides/user/pipeline/pipeline-templates/plan.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: single
+title:  "Visualize a Hydrated Pipeline"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+You can use `spin` CLI to visualize a pipeline created from a pipeline template
+without actually instantiating one.
+
+When you do this, you are creating JSON output that looks like regular pipeline
+JSON (plus the template reference and hydrated variables), but you're not
+creating the actual pipeline in Spinnaker.
+
+## Using `spin pt plan`
+
+```bash
+spin pipeline-templates plan --config <path to pipeline config>
```

Comment:
change to `--file`

Reply:
@jtk54 
/guides/user/spin/pipeline-templates/ says `--config`. Should it be `--file`?

Reply:
Yes, this is a recent change for 1.13.

Reply:
:+1: